ci-operator/config/openshift/multiarch-tuning-operator/openshift-multiarch-tuning-operator-main.yaml

```diff
@@ -21,6 +21,8 @@ images:
       as:
       - golang:1.21
   to: multiarch-tuning-operator
+- dockerfile_path: bundle.Dockerfile
+  to: multiarch-tuning-op-bundle
```

Comment:
so a bundle will be built for every pull request and after it merges it will be promoted?

Reply:
yeah, one is already built for in-pr tests, but it cannot be promoted. This can be kind of overwhelming (as it always refer the main image), but i prefer keeping it this way while we will do changes to it.. Moreover, this is a temporary solution that I would have to run openshift-tests-private and origin against a mixarch cluster with this operator installed until we can install the operator from the catalogs in the IIB.

Reply:
thanks a lot!

Reply:
I thought I sent a response for this. Yes, that bundle will always refer to the main operator image. This way, we can allow periodics outside of this repository config to install the operator until IIB will provide us the operator in the custom catalogs for testing.